kernal/vectors.s

```diff
@@ -187,7 +189,6 @@ jiobas	jmp iobase      ;return i/o base
 	.byte "MIST"
 
 	.segment "VECTORS"
-	.word nmi        ;program defineable
+	.word nmi     ;program defineable
```

Comment:
indent change?  I'm guessing not fully reverted.

Reply:
Will fix